test/splitgraph/conftest.py

```diff
@@ -609,3 +609,20 @@ def prepare_lq_repo(repo, commit_after_every, include_pk, snap_only=False):
             repo.commit(snap_only=snap_only)
     if not commit_after_every:
         repo.commit(snap_only=snap_only)
+
+
+@pytest.fixture
```

Comment:
Theoretically you could make this fixture session-scoped, since you're only ever reading from this schema so you don't need to reset it after every test. This should speed things up (you'd also have to use the `test_local_engine` fixture instead of `local_engine_empty`, since the latter is function-scoped).

Reply:
Makes sense, trying out now.